packages/enhanced-img/package.json

```
@@ -38,15 +38,16 @@
 	"dependencies": {
 		"magic-string": "^0.30.5",
 		"svelte-parse-markup": "^0.1.5",
-		"vite-imagetools": "^7.0.1"
+		"vite-imagetools": "^7.0.1",
+		"zimmerframe": "^1.1.2"
 	},
 	"devDependencies": {
 		"@types/estree": "^1.0.5",
 		"@types/node": "^18.19.48",
 		"estree-walker": "^3.0.3",
 		"rollup": "^4.14.2",
-		"svelte": "^4.2.10",
-		"typescript": "^5.3.3",
+		"svelte": "^5.0.0-next.0",
```

Should this be the official release now?
```suggestion
		"svelte": "^5.0.0",
```